block/types.go

```diff
@@ -8,6 +8,11 @@ import (
 )
 
 // TODO: move to types package
+
+const (
+	BatchOverhead = 700 // 500 block overhead + 117 for commit + 10 for batch overhead + safety margin
```

Comment:
not sure I understand how it was calculated.
also Is this in bytes? can we make it configurable? 
also maybe it can be a percentage/factor of the block size? vs hardcoded bytes?

Reply:
refactored it